### PR TITLE
Test oracle wallet using p12 keystore

### PR DESCRIPTION
**Pull request by KyleAure**
Ensure Oracle Wallet works with .p12 keystore.

### CODE REVIEW

File: dev/com.ibm.ws.jdbc_fat_oracle/publish/servers/com.ibm.ws.jdbc.fat.oracle.ssl/server.xml [hunk 1]
```diff
@@ -35,32 +35,36 @@
...
-    <dataSource id="oracleWallet" jndiName="jdbc/oracleWallet">
+    <dataSource id="oracleWalletSSO" jndiName="jdbc/oracleWalletSSO">
     	<jdbcDriver libraryRef="DBLib"/>
-    	<properties.oracle URL="${env.SSL_URL}" connectionProperties="${oracle.conn.props.wallet}"/>
+    	<properties.oracle URL="${env.SSL_URL}" connectionProperties="${oracle.conn.props}${oracle.wallet.sso}"/>
+    </dataSource>
+
+    <!-- Datasource using oracle wallet P12 auth and ssl connection -->
+    <variable name="oracle.wallet.p12"        value="oracle.net.wallet_location=${server.config.dir}/security/;oracle.net.wallet_password=${env.WALLET_PASS};" />
+
```
Comment (gjwatts) [inline dev/com.ibm.ws.jdbc_fat_oracle/publish/servers/com.ibm.ws.jdbc.fat.oracle.ssl/server.xml:48]: Suggestion for future update.  Could you replace:
```
<variable name="oracle.wallet.p12" value="oracle.net.wallet_location=${server.config.dir}/security/;oracle.net.wallet_password=${env.WALLET_PASS};" />
```
with
```
<variable name="oracle.wallet.p12" value="${oracle.wallet.sso}oracle.net.wallet_password=${env.WALLET_PASS};" />
```
?

**Review by gjwatts — APPROVED** [review]

File: dev/com.ibm.ws.jdbc_fat_oracle/publish/servers/com.ibm.ws.jdbc.fat.oracle.ssl/server.xml (hunk 1 above)
Comment (KyleAure) [reply dev/com.ibm.ws.jdbc_fat_oracle/publish/servers/com.ibm.ws.jdbc.fat.oracle.ssl/server.xml:48]: Yes, we could do that.  But I suspect this test bucket will be used by support to provide examples to customers.  As such, I'd rather be explicit in this case (properties needed for x configuration are....).

**Comment by KyleAure** [comment]
#build

**Comment by KyleAure** [comment]
#libby

**Comment by KyleAure** [comment]
#libby

**Pull request merged**